advanced-vector-search/flow-query.yml

```diff
@@ -1,20 +1,34 @@
 !Flow
 version: '1'
+env:
+  OMP_NUM_THREADS: ${{OMP_NUM_THREADS}}
 with:
   read_only: true
 pods:
   - name: encoder
     show_exc_info: true
     uses: yaml/encode.yml
-    parallel: $JINA_PARALLEL
+    shards: $JINA_PARALLEL
```

Comment:
```suggestion
    parallel: $JINA_PARALLEL
```

Reply:
I have mantainer `shards` because we have a conversation to maybe remove parallel for this concept as name so that I do not need to change in the future